packages/workbox-build/src/entry-points/options/defaults.js

```diff
@@ -7,18 +7,22 @@
 */
 
 module.exports = {
+  babelPresetEnvTargets: ['chrome >= 56'],
   cleanupOutdatedCaches: false,
   clientsClaim: false,
   globFollow: true,
   globIgnores: ['**/node_modules/**/*'],
   globPatterns: ['**/*.{js,css,html}'],
   globStrict: true,
   importWorkboxFrom: 'cdn',
-  injectionPointRegexp: /(precacheAndRoute\()\s*\[\s*\]\s*(\)|,)/,
+  injectionPoint: 'self.__WB_MANIFEST',
+  inlineWorkboxRuntime: false,
```

Comment:
I don't care much about this as long as there's an option, but since inlining everything is the default in Rollup (and not inlining requires loadz0r + amd, etc.), it feels to me like the default should be `true`.

I know not inlining potentially produces more cacheable builds, but I'm skeptical as to how much of a difference that'll actually make in practice since we update workbox frequently, and any configuration/logic change made by the user would likely change the workbox bundle anyway.

Reply:
As per our offline conversation, I *think* this is the right default, given that an entry in the precache manifest changing is a fairly likely event for any actively developed web app.

Paying the cost of re-downloading the Workbox runtime following every re-deploy doesn't seem like the right default if we could avoid it.

We could revisit the default if developers push back for some reason.